.gitignore

```diff
@@ -9,6 +9,9 @@
 # Production
 /build
 
+# MDB React Pro (Due to MDB React Pro license, we may not include the package in open-source projects)
```

Comment:
NOTE: Documentation, comments, and docstrings should be wrapped at 79 characters.

Reply:
I will remember that for future commits